frontend/src/utils/analysis-utils.ts

```diff
@@ -418,17 +418,24 @@ export class ExposedPopulationResult {
   legendText: string;
   statistic: AggregationOperations;
   tableData: TableRow[];
-  date: number;
+  analysisDate: ReturnType<Date['getTime']>;
   tableColumns: any;
 
   getTitle = (t?: i18nTranslator): string => {
     return t ? t('Population Exposure') : 'Population Exposure';
   };
 
-  getStatTitle = (t?: i18nTranslator): string => {
+  getLayerTitle = (t?: i18nTranslator): string => {
     return this.getTitle(t);
   };
 
+  getStatLabel(t?: i18nTranslator): string {
```

Comment:
I reviewed the code quickly and I think we can make `t` mandatory in all the getTitle and co functions. I can do it on this PR or in another one.

Reply:
(in our road to improving translation)

Reply:
Awesome, I like it. I'd be happy to make that change in this PR

Reply:
I just pushed that change but feel free to simplify further if beneficial